.github/workflows/branch_deployments.yml

```diff
@@ -6,18 +6,30 @@ env:
   DAGSTER_CLOUD_URL: https://pied-piper.dogfood.dagster.cloud
 
 jobs:
+  # Reads the list of locations to build from the cloud workspace file
```

Comment:
These are just example files, to copy as in a template, yes?

Reply:
yeah, examples/"tests" - will put up a corresponding change to the quickstart repo as well (that's the one people fork at the moment )